lib/components/signUp.dart

```diff
@@ -114,7 +115,7 @@ class _SignUpScreenState extends State<SignUpScreen> {
                   ),
                 ),
                 SizedBox(
-                  height: 10.0,
+                  height: MediaQuery.of(context).size.height * 0.01,
```

Comment:
It'd be great if you created a helper class for MediaQuery instead of calling `MediaQuery.of(context)` every time. This makes the code too redundant.